app/assets/stylesheets/users.scss

```diff
@@ -519,7 +519,6 @@
 .notification {
   border-radius: 4px;
   box-shadow: 0 1px 4px rgba(63, 63, 68, .15), 0 0 1px rgba(0, 0, 0, .5);
-  height: 78px;
```

Comment:
Why to remove this?

Reply:
cause notification was getting shrink 
![image](https://user-images.githubusercontent.com/86405648/213270016-5a97f187-3024-4a68-93b3-6ee2641fc287.png)